app/pathogen/arbovirus/dashboard/(map)/MapShadingLegend.tsx

```diff
@@ -0,0 +1,36 @@
+import { MapSymbology } from "@/app/pathogen/sarscov2/dashboard/(map)/map-config";
+import { Card, CardContent, CardHeader } from "@/components/ui/card";
+import { lastIndexOf } from "lodash";
+
+interface MapShadingLegendProps {
+    className?: string;
+  }
+
+
+export const MapShadingLegend = ({className}: MapShadingLegendProps) => {
+    const shadingLegendElements: {color: string, description: string}[] = [
+        {color: MapSymbology.CountryFeature.HasData.Color, description: "Seroprevalence estimates"},
+        {color: MapSymbology.CountryFeature.Default.Color, description: "No Seroprevalence estimates"}
+    ]
```

Comment:
I didn't see any "Non Applicable" color on map anywhere. So to me it only made sense for this legend to have two entries.